src/pudl/etl/epacems_assets.py

```diff
@@ -49,106 +43,72 @@ def _partitioned_path() -> Path:
     out=DynamicOut(),
     required_resource_keys={"dataset_settings"},
 )
-def get_years_from_settings(context):
+def get_year_quarters_from_settings(context):
     """Return set of years in settings.
 
     These will be used to kick off worker processes to process each year of data in
     parallel.
     """
     epacems_settings = context.resources.dataset_settings.epacems
-    years = {
-        EpaCemsPartition(year_quarter=yq).year for yq in epacems_settings.year_quarters
-    }
-    for year in years:
-        yield DynamicOutput(year, mapping_key=str(year))
+    for year_quarter in epacems_settings.year_quarters:
+        yield DynamicOutput(year_quarter, mapping_key=str(year_quarter))
 
 
 @op(
     required_resource_keys={"datastore", "dataset_settings"},
     tags={"memory-use": "high"},
 )
-def process_single_year(
+def extract_quarter(
     context,
-    year,
-    core_epa__assn_eia_epacamd: pd.DataFrame,
-    core_eia__entity_plants: pd.DataFrame,
-) -> YearPartitions:
+    year_quarter: str,
+) -> str:
     """Process a single year of EPA CEMS data.
```

Comment:
I think the docstring needs an update, both the one-line summary and the argument list (add `year_quarter`)